server/commands.py

```diff
@@ -11,33 +12,60 @@
 
 CommandHandler = Callable[["Session", list[str]], Awaitable[str | None]]
 
-command_handlers: dict[str, CommandHandler] = {}
 
+class Command:
+    def __init__(
+        self,
+        trigger: str,
+        callback: CommandHandler,
+        privileges: int | None = None,
+    ) -> None:
+        self.trigger = trigger
+        self.privileges = privileges
+        self.callback = callback
 
-def get_command_handler(trigger: str) -> CommandHandler | None:
-    return command_handlers.get(trigger)
 
+commands: dict[str, Command] = {}
```

Comment:
"commands" now store more information than just handlers, they also store privileges and the command trigger.